docparse/output_label_options_walkthrough.mdx

```diff
@@ -14,11 +14,11 @@ Let's look at the example below:
   <img src="/images/ntsb_incorrect_caption.png" />
 </Frame>
 
-You'll notice that the "Aviation Investigation Final Report" is incorrectly detected as a "Caption" here. To fix this, you can call `partition_file` with the `output_label_options` parameter:  
+You'll notice that the "Aviation Investigation Final Report" is incorrectly detected as a "Caption" here. To fix this when using the `aryn-sdk`, you can call `partition_file` with the `output_label_options` parameter:  
 
 
 ```python
-output_label_options = {"promote_title": True ,  "title_candidate_elements":["Section-header", "Caption"]}
+output_label_options = {"promote_title": True, "title_candidate_elements":["Section-header", "Caption"]}
 partitioned_file = partition_file(file, aryn_api_key, extract_images=True, extract_table_structure=True, use_ocr=False, selected_pages=[1] , output_label_options=output_label_options)
```

Comment:
Do we want to add line returns between each of the options so that it's more readable and customers don't have to scroll sideways to read this part of the documentation? Like this:

```python
output_label_options = {"promote_title": True, "title_candidate_elements":["Section-header", "Caption"]}
partitioned_file = partition_file(
    file, 
    aryn_api_key,
    extract_images=True,
    extract_table_structure=True,
    use_ocr=False,
    selected_pages=[1],
    output_label_options=output_label_options
)
```

Also, out of curiosity, can we simplify this example to the below? `use_ocr` is False by default, and I don't see how some of the other options are adding to the usefulness of this example.
```python
output_label_options = {"promote_title": True, "title_candidate_elements":["Section-header", "Caption"]}
partitioned_file = partition_file(
    file, 
    aryn_api_key,
    extract_table_structure=True,
    output_label_options=output_label_options
)
```

Reply:
sure